### PR TITLE
Updating the build scripts to support TLS 1.2 and paths with spaces in the name

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @livarcocc, @dotnet/dotnet-cli

### CODE REVIEW

File: build/build.ps1 [hunk 1]
```diff
@@ -22,6 +22,7 @@ Param(
 
 Set-StrictMode -Version 2.0
 $ErrorActionPreference = "Stop"
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```
Comment (nguerrera) [inline build/build.ps1]: The fix I saw in C# was to `|` Tls12 on to the defaults. Should we do that here?

**Review by nguerrera — APPROVED** [review]

File: build/build.ps1 (hunk 1 above)
Comment (tannergooding) [reply build/build.ps1]: I copied this line verbatim from @jaredpar's fix.

Comment (tannergooding) [reply build/build.ps1]: https://github.com/dotnet/roslyn/pull/25053/files

Comment (jaredpar) [reply build/build.ps1]: I copied from @tmat 

https://github.com/dotnet/roslyn-tools/pull/169

Comment (tmat) [reply build/build.ps1]: I guess we can OR it.

**Pull request merged**